### DISCUSSION TITLE
lazygit with yadm

### DISCUSSION THREAD

**Discussion by nadir-ishiguro** [LazyVim/LazyVim#917] [Q&A]
Hi,
I'm trying to add a lazygit mapping that's identical to the ones included with LazyVim but that uses the yadm repository.

Currently, I'm using an alias in another terminal for this:

`alias yl='lazygit --use-config-file "$HOME/.config/yadm/lazygit.yml,$HOME/Library/Application Support/lazygit/config.yml" --work-tree ~ --git-dir ~/.local/share/yadm/repo.git'`

I got that from [this lazygit issue.](https://github.com/jesseduffield/lazygit/discussions/1201).

The call to  `~/.config/yadm/lazygit.yml:` is explained in the file's comment:

```
keybinding:
  files:
    toggleStagedAll: '<c-a>' # avoid "stage/unstage all" since it hangs lazygit, cause I guess it tries to list all files
```

I feel this shouldn't be too difficult but I'm not getting anywhere.

**Comment by andresatierf** [comment]
I'm also having this problem
